ray-operator/README.md

```diff
@@ -68,9 +68,9 @@ There are three example config files to deploy RayClusters included here:
 
 Sample  | Description
 ------------- | -------------
-[ray-cluster.mini.yaml](config/samples/ray-cluster.mini.yaml)   | Small example consisting of 1 head pod.
-[ray-cluster.heterogeneous.yaml](config/samples/ray-cluster.heterogeneous.yaml)  | Example with heterogenous worker types. 1 head pod and 2 worker pods, each of which has a different resource quota.
-[ray-cluster.complete.yaml](config/samples/ray-cluster.complete.yaml)  | Shows all available custom resouce properties.
+[ray-cluster.mini.yaml](../../ray-operator/config/samples/ray-cluster.mini.yaml)   | Small example consisting of 1 head pod.
```

Comment:
wait. `config/samples/ray-cluster.complete.yaml` seems the right relative path. 

![image](https://user-images.githubusercontent.com/4739316/173704084-2d7493ff-74d9-4024-b0a4-16c1678e0d82.png)

I tried to click the link and it works perfectly. Could you double check your change?